.env.example

```diff
@@ -16,6 +16,13 @@ EMAIL_SERVER_PASSWORD=
 EMAIL_SERVER_HOST=
 EMAIL_SERVER_PORT=
 EMAIL_FROM=
+EMAIL_TO=
+EMAIL_TO_AVOCADO=
+EMAIL_TO_BANANA=
+EMAIL_TO_CARROT=
+EMAIL_TO_CHEATSHEET=
+EMAIL_TO_SWEET_POTATO=
+EMAIL_TO_WATERMELON=
```

Comment:
I figured it'd be a good idea to keep people's emails out of the repo (also good for changing them if we need to) 😁